python/tests/test_metrics.py

```diff
@@ -23,6 +23,15 @@
 )
 from seldon_core.user_model import client_custom_metrics
 
+import seldon_core
+
+
+def test_parametrize(client_gets_metrics):
```

Comment:
We can probably remove this one before merging. That's a nice way to test parametrised fixtures!

Reply:
hmm... tests bellow depends on value returned `client_gets_metrics`... I will maybe just rename it to be a test of a fixture?

Reply:
Why would we want to have tests of fixtures? Isn't that like a metatest?